server/routes/api/users.js

```diff
@@ -0,0 +1,101 @@
+const express = require("express");
+const jwt = require("jsonwebtoken");
+const bcrypt = require("bcrypt");
+
+const User = require("../../models/User");
+const { passportSecret } = require("../../config/keys");
+const validateEmail = require("../../validation/email");
+const validatePassword = require("../../validation/password");
```

Comment:
Did you know if there was a way to use absolute paths from the server route directory easily? I'm not so familiar with an easy way in express. If there is no easy way, you can use relative paths like you have done here

Reply:
from what i can find, it looks like relative paths are the default by design. there are a few ways like modifying the NODE_PATH, placing the files in node_modules, or constructing the system absolute path and using the native absolute path require but they seem kinda hacky.